queries.sql

```diff
@@ -6,3 +6,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- update the animals table by setting the species column to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon
+UPDATE animals SET species = 'unspecified' WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```

Comment:
- [ ] You are still missing one transaction as shown below. Please add it
![Screenshot 2023-06-13 at 15 20 10 (2)](https://github.com/Bellagirl-maker/vet-clinic/assets/64774936/6f70cbc0-2989-4cfd-9fc7-cb5053e7d656)